content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -522,24 +522,29 @@ This means that support is still available but will be removed in a future Kuber
 * [`rbd`](/docs/concepts/storage/volumes/#rbd) - Rados Block Device (RBD) volume
   (**deprecated** in v1.28)
 
-Older versions of Kubernetes also supported the following in-tree PersistentVolume types:
+The following in-tree PersistentVolume types are no longer available, but are supported through `CSIMigration` and have a corresponding CSI driver implemented.
 
 * [`awsElasticBlockStore`](/docs/concepts/storage/volumes/#awselasticblockstore) - AWS Elastic Block Store (EBS)
-  (**not available** in v1.27)
+  (**not available** starting with v1.27, see [AWS EBS](https://github.com/kubernetes-sigs/aws-ebs-csi-driver) CSI driver)
```

Comment:
Instead of "not available", I would say that these require the CSIMigration feature gate + CSI driver to be installed.

Reply:
```suggestion
  &mdash;this support is **only** available via a separate CSI driver for Kubernetes version 1.27 and onwards,
  as Kubernetes' core no longer provides built-in support. The CSI driver you need to install and configure is
  [AWS EBS](https://github.com/kubernetes-sigs/aws-ebs-csi-driver).
```